CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 ## CHANGE LOG
 
+### Version 1.7.0 *(7th June 2023)*
```

Comment:
@shivamsharma2710 change to today's date

Reply:
This is done.